### PR TITLE
[blog] Add post detailing base plans refresh 2019-01-22

### PR DESCRIPTION
**Pull request by smacfarlane**
This adds a blog post about the upcoming base-plans refresh.

Signed-off-by: Scott Macfarlane <smacfarlane@chef.io>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

**Comment by smacfarlane** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/480599379).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports acceptance has been deployed](https://travis-ci.org/habitat-sh/habitat/builds/480599379).

I love new and shiny websites!

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

File: www/source/blog/2019-01-22-base-plans-refresh.html.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+
+We also perform periodic base plan refreshes to make it easy for users of Habitat to receive the latest performance improvements, security patches, and bug fixes from the underlying software libraries that power your application. Our periodic base plan refresh means you will receive these updates automatically, and you will never worry about your application being tied to an aging Operating System version.
+
+This refresh contains OpenSSL version 1.0.2q and brings two significant improvements: the CRIME vulnerability is mitigated by default and FIPS mode is available to all users. FIPS mode is particularly interesting; although it is not enabled by default it can be activated at runtime simply by setting the environment variable `OPENSSL_FIPS=1` - this ensures that any application that is built on OpenSSL can automatically operate in a FIPS compliant manner.
+
+This base plan refresh upgrades GCC from version 7.3.0 to version 8.2.0. GGC 8 contains a number of performance improvements that affect every application built with the base plans. [https://gcc.gnu.org/gcc-8/changes.html](https://gcc.gnu.org/gcc-8/changes.html)
```
Comment (johnjelinek) [inline www/source/blog/2019-01-22-base-plans-refresh.html.md]: `s/GGC 8/GCC 8/`

**Pull request merged**